File: Recordings.vb


Public Class Recording
    Public Property url As String
    Public Property id As Integer
    Public Property label As String
    Public Property camera As Camera
    Public Property description As String
    Public Property timestamp As DateTime
    Public Property duration As Integer
    Public Property status As Integer
    Public Property m3u8 As String
    Public Property thumbnails As String()
    Public Property classification As Integer
    Public Property created_at As DateTime
    Public Property updated_at As DateTime
    Public Property mp4 As String
    Public Property events As Object
End Class

Public Class Recordings
    Public Property [next] As Object
    Public Property previous As Object
    Public Property results As Recording()
End Class

File: Recordings.cs

using System;
using System.Collections.Generic;
using System.Diagnostics;
using System.Globalization;
using System.IO;
using System.Linq;
using System.Reflection;
using System.Runtime.CompilerServices;
using System.Security;
using System.Text;
using System.Threading.Tasks;
using Microsoft.VisualBasic;

public class Recording
{
    public string url { get; set; }
    public int id { get; set; }
    public string label { get; set; }
    public Camera camera { get; set; }
    public string description { get; set; }
    public DateTime timestamp { get; set; }
    public int duration { get; set; }
    public int status { get; set; }
    public string m3u8 { get; set; }
    public string[] thumbnails { get; set; }
    public int classification { get; set; }
    public DateTime created_at { get; set; }
    public DateTime updated_at { get; set; }
    public string mp4 { get; set; }
    public object events { get; set; }
}

public class Recordings
{
    public object next { get; set; }
    public object previous { get; set; }
    public Recording[] results { get; set; }
}
